### DISCUSSION TITLE
Joomla custom fields support

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#88] [Ideas]
I was thinking that using Joomla custom fields in JoomCCK fields type will be a good idea.
but i think will be very hard to implement.
Maybe i will try in future to do it.
This feature will help to use Joomla custom fields
But the main issues that many features not yet supported , like filtering, import/export sorting ... in Joomla custom fields which means JoomCCK fields are still very advanced
So i will think about it carefully.

any suggestions about this feature are welcome.

**Comment by Giorgi625** [comment]
yes JoomCCK fields are much better. When Sergey gave up on Cobalt I tried to move to Joomla content with its custom fields but it is very weak compared to JoomCCK, so I think better as it is now. 
Just maybe improve some fields with adding features, but later when JoomCCK will be stable. 

For me most important currently is to get stable version asap and we can think about improving later, but if like there are improvements which should be done now to avoid problems later then lets do it.

**Reply by andiamine** [reply]
Yes, i think maybe we do default integration (filling and displaying fields) and once joomla add a good filtering system will be easy to continue deep integration. but for now main Goal is to make JoomCCK stable.

I think the main improvement that should be done is App mode, will be easy for users to create sections and manage sections. current mode is very hard and things are not well related.
